Homebrew/Usage.md

```diff
@@ -1,30 +1,52 @@
 # Using Homebrew
+
 To install a package (or **Formula** in Homebrew vocabulary) simply type:
 
     $ brew install <formula>
 
-To update Homebrew's directory of formulae, run:
+To update Homebrews directory of formulas, run:
```

Comment:
"Homebrew's"

Reply:
Homebrew themselves use "formulae"

(also some other "formulas" below)

Reply:
Yes, you're right 👌